### ISSUE TITLE
[BUG] cellfinder-napari doesn't work with pyqt5

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/cellfinder-napari#134]
Opening a cellfinder-napari widget with pyqt5 gives the error:
```
TypeError: setMinimumWidth(self, int): argument 1 has unexpected type 'float'
```

I've reproduced this on Windows and macOS (M1). Removing pyqt5 and installing pyside2 seems to fix the issue (at least on windows).

The issue doesn't seem to be present for other BrainGlobe napari plugins. 

image.sc issue for reference: https://forum.image.sc/t/cannot-launch-cellfinder-napari-plugin/71114/4

**Comment by adamltyson** [comment]
Curation seems fine (this widget wasn't updated recently).

**Comment by adamltyson** [comment]
Seems to be specific to Python 3.10

**Comment by adamltyson** [comment]
I can only guess that it was `0.5.1` based on the [release dates](https://pypi.org/project/magicgui/0.5.1/#history).

**Comment by dstansby** [comment]
Fab, I can reproduce with `magicgui=0.5.1`, and confirm this was fixed in `magicgui=0.6.0`. So probably fine to close this issue?

**Comment by adamltyson** [comment]
Sounds good. Thanks for digging into this!
